src/EventListener/InitializeSystemListener.php

```diff
@@ -151,6 +167,9 @@ private function boot($routeName, $basePath)
         // Log PHP errors
         $this->iniSet('error_log', $this->rootDir . '/system/logs/error.log');
 
+        // Support legacy Session access
+        $this->initializeLegacySessionAccess();
```

Comment:
I can, I just saw that it is now before `Config::preload();` which means that if somebody did something like
```php
$_SESSION['FE_DATA']['foo'] = 'bar';
```
in his `localconfig.php` it would still work. If I move it down, it won't work anymore.